### PR TITLE
use safari inapp browser if available

### PR DESCRIPTION
**Pull request by johnthepink**
tries to use the safari in app browser. falls back to other in app browser if ios 8 or android. need to see if something similar for android

sister PR in app

### CODE REVIEW

File: core/util/inAppLink.jsx [hunk 1]
```diff
@@ -1,9 +1,40 @@
+function openUrl(url) {
+  SafariViewController.isAvailable(function (available) {
```
Comment (jbaxleyiii) [inline core/util/inAppLink.jsx]: @johnthepink we should probably wrap this with either an existence check on `SafariViewController` or a more general `try/catch`

Comment (johnthepink) [inline core/util/inAppLink.jsx]: @jbaxleyiii I did think about this, but I saw this on their end:

https://github.com/EddyVerbruggen/cordova-plugin-safariviewcontroller/commit/19266d46347ef45f5cdf2955a00d5f05a128ca57

You are probably right to be safe though, could possibly not load or something.

Comment (johnthepink) [inline core/util/inAppLink.jsx]: actually. totally makes sense in this repo. thought i was in the app repo

Comment (johnthepink) [inline core/util/inAppLink.jsx]: err. well. it is wrapper in a `Meteor.isCordova`, so maybe you were thinking in both? @jbaxleyiii

**Comment by johnthepink** [comment]
@jbaxleyiii sorry for all the comments. you were right. added an existence check and a try block.

**Pull request merged**